ui/app/templates/components/clients/error.hbs

```diff
@@ -0,0 +1,25 @@
+<EmptyState
+  @title={{if (eq @error.httpStatus 403) "You are not authorized" "Error"}}
+  @subTitle={{concat "Error " @error.httpStatus}}
```

Comment:
Just a heads up that the `concat` helper is fine but you should be able to do
`@subTitle="Error {{@error.httpStatus}}"`

Reply:
Good catch! I think I had an `if` there and forgot to pull out the `concat`